### PR TITLE
[US-20] Update examples with new builtin helper functions

### PR DESCRIPTION
**Pull request by kellemNegasi**
PR for https://unidoc.atlassian.net/browse/US-20

### CODE REVIEW

File: templates/bank-account-statement/pdf_bank_account_statement.go [hunk 1]
```diff
@@ -31,7 +29,7 @@ func init() {
 		panic(err)
 	}
 
-	common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
+	// common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
```
Comment (sampila) [inline templates/bank-account-statement/pdf_bank_account_statement.go:32]: Remove commented code

**Review by sampila — CHANGES_REQUESTED** [review]
Overall looks good, added minor comment

**Comment by 3ace** [comment]
since the new helper function would be included a new unipdf release, I think this PR should be put in draft mode first. Otherwise this updated example would not working correctly.

After there's a new unipdf release, you could reopen the PR and don't forget to update dependency to use the latest unipdf version.

**Review by sampila — APPROVED** [review]
Looks good to me

**Review by 3ace — APPROVED** [review]
lgtm

**Pull request merged**